src/components/AvatarShape.js

```diff
@@ -31,6 +33,12 @@ const Image = styled('img')`
   position: absolute;
   top: 0;
   left: 0;
+  transition: filter ${TRANSITION_STYLE};
+  ${({$greyscale}) =>
+    $greyscale &&
+    `
+      filter: grayscale(1);
+  `}
```

Comment:
I can't see a way of doing this with styled-system without installing [the css package](https://styled-system.com/css) 🤔 is this okay?

Reply:
This is fine, although I think if you change the name to `hover` it's a bit easier to understand what's happening IMO

```scss
  filter: grayscale(1); // makes it grayscale by default

  ${({hover}) => hover && `filter: grayscale(0);` } // turn it off when hover is enabled for full colour gifs
```